src/types/featureFlags.ts

```diff
@@ -1,8 +1,11 @@
 import { FEATURE_FLAGS } from "constants/featureFlags"
 
+export type PrivatisationWhitelist = { repos: string[] }
+
 // Example usage: const gb = useGrowthBook<FeatureFlags>();
 export interface FeatureFlags {
   [FEATURE_FLAGS.STYLING_REVAMP]: boolean
+  [FEATURE_FLAGS.REPO_PRIVATISATION]: PrivatisationWhitelist
```

Comment:
Hmm why are we not using a boolean value like styling revamp? This way we are exposing info to the client on which sites use this feature, which seems something that we can avoid by switching to boolean type instead, would this be feasible?

Reply:
We want to gate the rollout slowly. See Slack convo for more info

Reply:
Hmm correct me if I am wrong ya but we can still rollout per site basis without returing the entire list to the client right? eg here:
![Screenshot 2023-09-27 at 5 27 08 PM](https://github.com/isomerpages/isomercms-frontend/assets/42832651/d819402a-78cb-468c-975b-5be056e4c7e0)

Reply:
Oh that's what you meant! I wanted to be explicit, but actually you have a very good point. We can off-load the conditional part to growthbook! Let me try this approach 👍🏼

Reply:
fixed